pkg/backup/restore/restore_manager.go

```diff
@@ -135,9 +148,16 @@ func (rm *restoreManager) makeImportJob(restore *v1alpha1.Restore) (*batchv1.Job
 	ns := restore.GetNamespace()
 	name := restore.GetName()
 
-	envVars, reason, err := backuputil.GenerateTidbPasswordEnv(ns, name, restore.Spec.To.SecretName, restore.Spec.UseKMS, rm.deps.KubeClientset)
-	if err != nil {
-		return nil, reason, err
+	var (
+		envVars []corev1.EnvVar
+		reason  string
+		err     error
+	)
+	if restore.Spec.To != nil {
```

Comment:
We have removed the GC logic in `cmd/backup-manager/app/import/manager.go`, should we also remove this?